src/comestible_inv_listitem.cpp

```diff
@@ -0,0 +1,771 @@
+#include "auto_pickup.h"
+#include "avatar.h"
+#include "cata_utility.h"
+#include "catacharset.h"
+#include "input.h"
+#include "item_category.h"
+#include "item_search.h"
+#include "item_stack.h"
+#include "options.h"
+#include "output.h"
+#include "player.h"
+#include "player_activity.h"
+#include "string_formatter.h"
+#include "color.h"
+#include "int_id.h"
+#include "item.h"
+#include "ret_val.h"
+#include "type_id.h"
+#include "enums.h"
+#include "faction.h"
+#include "material.h"
+#include "comestible_inv_listitem.h"
+
+#include <algorithm>
+#include <cassert>
+#include <cstring>
+#include <string>
+#include <vector>
+#include <initializer_list>
+#include <iterator>
+#include <utility>
+#include <numeric>
+
+void comestible_inv_listitem::print_columns( std::vector<comestible_inv_columns> columns,
+        comestible_select_state selected_state, catacurses::window window, int right_bound,
+        int cur_print_y )
+{
+    is_selected = selected_state;
+    int w;
```

Comment:
Should be defined inside of the loop. It's never used outside of it. Apples to the below variables as well.